src/main/java/org/embulk/output/mailchimp/MailChimpClient.java

```diff
@@ -109,14 +115,14 @@ public void handleErrors(List<ErrorResponse> errorResponses)
      * @return the map
      * @throws JsonProcessingException the json processing exception
      */
-    public Map<String, Map<String, InterestResponse>> extractInterestCategoriesByGroupNames(final PluginTask task)
+    public Map<String, Map<String, InterestResponse>> extractInterestCategoriesByGroupNames(final PluginTask task,
+                                                                                            Schema schema)
             throws JsonProcessingException
     {
         try (MailChimpRetryable retryable = new MailChimpRetryable(task)) {
             Map<String, Map<String, InterestResponse>> categories = new HashMap<>();
+            List<String> taskGroupingColumns = task.getGroupingColumns().get();
```

Comment:
getGroupingColumns is optional, look like the line of code will throw error when it is not presented since the condition to check whether it is optional is below.

Reply:
@vinhvd thanks, fixed